### DISCUSSION TITLE
Dịch Singular Value Decomposition

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#58] [Terminology] [answered]
Từ này mình thấy đang được dịch là `phân rã giá trị đơn`, sửa lại thành `phân rã giá trị suy biến` cho giống với `ma trận suy biến - singular matrix` được không nhỉ?

cc @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
Có ai có tham khảo dịch ở mấy sách kinh điển không nhỉ?

**Reply by tiepvupsu** [reply]
Sách MLCB của mình dịch là "Phân tích giá trị suy biến".

**Reply by ngcthuong** [reply]
Cho mình hỏi chút là tại sao lại là "suy biến" nhỉ? @tiepvupsu

**Reply by tiepvupsu** [reply]
Mình cũng không phải là người sáng tác ra từ này. Nhiều tài liệu tiếng Việt dùng non-singular như là "không suy biến" và "singular matrix" như là ma trận suy biến.

**Reply by lkhphuc** [reply]
Tới giờ em vẫn không biết tại sao tiếng anh lại gọi là `singular`, trong khi `suy biến` thì dễ hiểu hơn là do có các vector phụ thuộc tuyến tính nên số phương trình thực sẽ bị giảm xuống.

**Reply by hnguyentt** [reply]
Theo thiển ý của em gọi `singular matrix` có thể hiểu theo 2 hướng sau:
* Nó là một ma trận "cô đơn, lẻ bóng" ví nó không có ma trận nghịch đảo, nó ko có couple với ai hết: https://mathworld.wolfram.com/SingularMatrix.html
* Khi lấy bất kỳ một ma trận `NxN` thì thông thường nó sẽ là ma trận khả nghịch,  chỉ trong trường hợp đặc biệt mới là ma trận bất khả nghịch. Từ `singular` trong nghĩa này nghĩa là `kỳ lạ, bất thường` ("odd", "exceptional", "unusual", "peculiar"). Theo giải thích ở đây: https://math.stackexchange.com/questions/42649/why-are-invertible-matrices-called-non-singular?newreg=1ccc462f34144db5bae9b8f1c7b38fb6

**Reply by minhduc0711** [reply]
Theo option 1 thì mình cũng bị singular 😞

**Reply by lkhphuc** [reply]
Vậy thì `đơn` là không đúng rồi. Chốt như a Tiệp là `suy biến` nhé.

**Reply by ngcthuong** [reply]
Từ này đã dùng nhiều rồi nên mình dịch như cũ thôi là "suy biến" nhé tất cả thanh niên đã/đang/ và sẽ hết singular?

**Comment by thanhcsf** [comment, marked as answer]
Vậy chốt là **Phân tích giá trị suy biến** nha anh em.
